client/displayservers/Wayland/shell_libdecor.c

```diff
@@ -51,28 +51,32 @@ static void libdecorHandleError(struct libdecor * context, enum libdecor_error e
 static void libdecorFrameConfigure(struct libdecor_frame * frame,
     struct libdecor_configuration * configuration, void * opaque)
 {
+  if (wlWm.needsResize)
+    return;
```

Comment:
Is this correct? If there are two configure events in a row, the latter should have precedence. We should at least update `wlWm.width` and `wlWm.height` so that the EGL viewport has the correct size when we resize it.

Reply:
I found if the previous resize hadn't been handled yet then when drag was shrinking the window it leads to the renderer attempting to draw on a larger surface than is available. May not be 100%, but I haven't been able to crash it in testing.

Reply:
Scratch that - seems to work fine without it provided win:forceAspect is disabled.
With forceAspect enabled there's more crashes but that's likely down to the race between frame updates from the cursor and from the timer.